android-json-form-wizard/src/main/java/com/vijay/jsonwizard/event/ExpansionPanelInflateEvent.java

```diff
@@ -0,0 +1,37 @@
+package com.vijay.jsonwizard.event;
+
+import android.widget.LinearLayout;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExpansionPanelInflateEvent extends BaseEvent {
```

Comment:
This class looks almost identical to `RefreshExpansionPanelEvent`. 

Wouldn't a better approach be to factor out the similar components to the `BaseEvent`?